src/main/webapp/WEB-INF/tags/answerWith.tag

```diff
@@ -7,7 +7,7 @@
 <%@attribute name="commentVotes" type="org.mamute.model.CommentsAndVotes" required="true" %>
 <section class="post-area ${answer.isVisibleForModeratorAndNotAuthor(currentUser.current) ? 'highlight-post' : '' }">
 	<div class="post-meta">
-		<tags:voteFor item="${answer}" type="${t['answer.type_name']}" vote="${vote}"/>
+		<tags:voteFor item="${answer}" type="${answer.typeName}" vote="${vote}"/>
```

Comment:
There are other places like in questionWith.tag:
```
<tags:watchFor watchable="${question}" type="${t['question.type_name']}"/>
```
I don't think there's anything we can remove here. The problem only showed with the votes, because key was double-translated (the first part of the translation key was already translated).